tests/test_cli.py

```diff
@@ -12,16 +12,16 @@
 
 from wetterdienst import cli
 
-# Individual settings for observations and forecasts
+# Individual settings for observation and forecast
 SETTINGS_STATIONS = [
-    "observations stations --resolution=daily --parameter=kl --period=recent",
-    "forecasts stations",
+    "observation stations --resolution=daily --parameter=kl --period=recent",
+    "forecast stations --parameter=large",
 ]
 
 SETTINGS_READINGS = [
-    "observations values --resolution=daily --parameter=kl --period=recent "
+    "observation values --resolution=daily --parameter=kl --period=recent "
     "--date=2020-06-30",
-    f"forecasts values --mosmix-type=large --parameter=DD "
+    f"forecast values --parameter=DD --mosmix-type=large"
```

Comment:
Thank you for keeping the possibility to request the relevant parameters using their uppercase representations.